src/index.ts

```diff
@@ -77,6 +80,27 @@ const getPoints = async (argv: Arguments): Promise<Point[]> => {
     result.push(...points);
   }
 
+  if (argv.services.includes("mastodon")) {
+    log("mastodon (current): starting");
+    const points = await mastodon.getCurrentPoints();
+    log("mastodon (current): finished");
+    result.push(...points);
+  }
+
+  if (argv.services.includes("linkedin")) {
+    log("linkedin (current): starting");
+    const points = await linkedin.getCurrentPoints();
+    log("linkedin (current): finished");
+    result.push(points);
+  }
+
+  if (argv.services.includes("instagram")) {
+    log("instagram (current): starting");
+    const points = await instagram.getCurrentPoints();
```

Comment:
This should take the env vars as an argument, rather than reading them from `process.env` directly in `instagram.ts`. See the function `setUpInflux()` as an example.